demos/asp-net-core/Divergent.Sales.API.Host/Controllers/OrdersController.cs

```diff
@@ -36,13 +35,12 @@ public IEnumerable<dynamic> Get(int pageIndex, int pageSize)
             {
                 var orders = db.Orders
                     .Include(o => o.Items)
-                    .OrderBy(o => o.Id) //required by SQLite EF
+                    .OrderBy(o => o.OrderNumber) //required by SQLite EF
```

Comment:
Is the comment still relevant?

Reply:
For the sake of the sample/demo sorting is not required, it’s the EF implementation that requires sorting. He comment is there simply to remind that, we can remove it without any issue.